man/systemd-nspawn.xml

```diff
@@ -220,6 +220,18 @@
         <xi:include href="version-info.xml" xpointer="v226"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--cleanup</option></term>
+
+        <listitem>
+          <para>Clean up directories used by the container, and exit without invoking any containers. This
```

Comment:
maybe say "left-over mounts and underlying mount points" rather than "directories"